platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+            # Align the second structure to the first
+            aligned_pdb2_path = align_structures(pdb1_path, pdb2_path)
+            
+            # Create three separate views
+            view1 = create_structure_view(pdb1_path, 'blue', label="AlphaFold2")
+            view2 = create_structure_view(aligned_pdb2_path, 'darkgrey', label="ESMFold")
+            
+            # Create combined view
+            view3 = py3Dmol.view(width=800, height=400)
+            
+            # Load and display first structure (AlphaFold2)
+            with open(pdb1_path, 'r') as f:
+                pdb1_data = f.read()
+            view3.addModel(pdb1_data, "pdb")
+            view3.setStyle({'model': -1}, {'cartoon': {'color': 'blue'}})
+            
+            # Load and display aligned second structure (ESMFold)
+            with open(aligned_pdb2_path, 'r') as f:
+                pdb2_data = f.read()
+            view3.addModel(pdb2_data, "pdb")
+            view3.setStyle({'model': 1}, {'cartoon': {'color': 'darkgrey'}})
+            
+            # Set up the combined view
+            view3.zoomTo()
+            
+            # Add labels for combined view
+            view3.addLabel("AlphaFold2", {'position': {'x': -20, 'y': 0, 'z': 0}, 'backgroundColor': 'blue', 'fontColor': 'white'})
+            view3.addLabel("ESMFold", {'position': {'x': 20, 'y': 0, 'z': 0}, 'backgroundColor': 'darkgrey', 'fontColor': 'white'})
+            
+            return view1, view2, view3        
+        ```
+
+    1. Display interactive 3D structure views: 
+
+        ```python 
+        # Visualize the structures
+        view1, view2, view3 = visualize_structures(alphafold2_multimer_standard, esmfold_multimer)
+
+        # Display all views
+        print("AlphaFold2 Structure:")
+        view1.show()
+        print("\nESMFold Structure:")
+        view2.show()
+        print("\nAligned Structures:")
+        view3.show()        
+        ```
+
+</details>    
+
+### RStudio: Analyze RNASeq data and differential expression statistics 
+
+Studios and RStudio notebooks enable interactive analysis using R libraries and tools. For example, Shiny for R enables you to render functions in a reactive application and build a custom user interface to explore your data. The public data used in this section consists of RNA sequencing data that was processed by the **nf-core/rnaseq** pipeline to quantify gene expression, followed by **nf-core/differentialabundance** to derive differential expression statistics. This section demonstrates how to create a studio to perform further analysis with these results from cloud storage. One of these outputs is an RShiny application that can be deployed for interactive analysis.
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment with at least 2 CPUs available, create one with the following attributes:
```

Comment:
Same comment about starting sentence with "Studios and". I suspect we'll see this in all sections so I'll stop commenting beyond this second entry.